src/test/resources/com/owlcyberdefense/mil-std-2045/milstd2045.tdml

```diff
@@ -696,6 +696,7 @@ operator_reply_request_indicator set to 1">
                   <value>OK_VMF</value>
                 </umf>
                 <message_standard_version>
+                  <vmf_hdr_C>OK_6017A</vmf_hdr_C>
```

Comment:
This change to the elements is unavoidable. This will require expected test infosets to change for all uses of mil-std-2045 headers with VMF. 
This element varies. For rev C headers it is vmf_hdr_C, for D1 it is vmf_hdr_D1.